net/openssh/Makefile

```diff
@@ -165,6 +173,12 @@ CONFIGURE_ARGS += \
 	--without-kerberos5 \
 	--with-stackprotect \
 	--with$(if $(CONFIG_OPENSSL_ENGINE),,out)-ssl-engine
+	
+
+ifdef CONFIG_OPENSSH_LIBFIDO2
```

Comment:
A similar trick can be applied as with CONFIG_OPENSSL_ENGINE above to reduce this to one line.